### PR TITLE
Add ref to CoC and other things

### PR DESCRIPTION
**Pull request by duglin**
Wait until https://github.com/cloudevents/spec/pull/663 is resolved.

However, look over the wording and once we get that right I'll open similar PRs in other SDK repos

Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,3 +131,14 @@ There you will find Express.js, TypeScript and Websocket examples.
...
+
+While there are some SDK specific processes defined, most governance related
+material can be found in the
+[CloudEvents `community`](https://github.com/cloudevents/spec/tree/master/community)
+directory. In particular, in there you will find information concerning
+how SDK projects are
+[managed](https://github.com/cloudevents/spec/blob/master/community/SDK-GOVERNANCE.md),
+[guidelines](https://github.com/cloudevents/spec/blob/master/community/SDK-maintainer-guidelines.md)
+for how PRs are reviewed, and our
+[Code of Conduct](https://github.com/cloudevents/spec/blob/master/community/GOVERNANCE.md#additional-information)
```
Comment (lance) [inline README.md:143]: Should this just link directly to the CNCF CoC?

File: README.md [hunk 2]
```diff
@@ -131,3 +131,14 @@ There you will find Express.js, TypeScript and Websocket examples.
...
 for information on how to get involved.
+
+While there are some SDK specific processes defined, most governance related
+material can be found in the
+[CloudEvents `community`](https://github.com/cloudevents/spec/tree/master/community)
+directory. In particular, in there you will find information concerning
+how SDK projects are
+[managed](https://github.com/cloudevents/spec/blob/master/community/SDK-GOVERNANCE.md),
+[guidelines](https://github.com/cloudevents/spec/blob/master/community/SDK-maintainer-guidelines.md)
+for how PRs are reviewed, and our
```
Comment (lance) [inline README.md:142]:
```suggestion
for PR reviews and approval, and our
```

File: README.md [hunk 3]
```diff
@@ -131,3 +131,14 @@ There you will find Express.js, TypeScript and Websocket examples.
 We love contributions from the community! Please check the
 [Contributor's Guide](https://github.com/cloudevents/sdk-javascript/blob/master/CONTRIBUTING.md)
 for information on how to get involved.
+
+While there are some SDK specific processes defined, most governance related
```
Comment (lance) [inline README.md:135]:
```suggestion
Each SDK may have its own unique processes, tooling and guidelines, common governance related
```

**Comment by lance** [comment]
@duglin ping - just making sure this doesn't fall off of your radar

File: README.md (hunk 1 above)
Comment (duglin) [reply README.md:164]: I decided for the indirect ref in the hopes that if we produce our own CoC one day then we'd only need to change the spec's repo and not each individual SDK's.  But if you feel we should point directly to CNCF's I can change it.

File: README.md (hunk 2 above)
Comment (duglin) [reply README.md:163]: done

File: README.md (hunk 3 above)
Comment (duglin) [reply README.md:156]: done

**Comment by duglin** [comment]
thanks for the reminder @lance 
Give it another look-over....

**Comment by lance** [comment]
@duglin looks good to me. The title still has this as WIP. Is that the case?

**Comment by duglin** [comment]
oops - fixed!

**Pull request merged**